OpenRA.Mods.Common/Pathfinder/PathSearch.cs

```diff
@@ -57,40 +57,33 @@ public static IPathSearch Search(World world, Locomotor locomotor, Actor self, B
 
 		public static IPathSearch FromPoint(World world, Locomotor locomotor, Actor self, CPos @from, CPos target, BlockedByActor check)
```

Comment:
`FromPoint` seems to be a bit pointless now that it is basically just directing to `FromPoints` without any logic of it's own. Would there be any objection to replacing all references to `FromPoint` with `FromPoints` and removing it?

Reply:
It harbors the important logic of creating the `froms` array! :wink:

Reply:
If we dropped this we'd have to create a single-entry array in every place that currently uses `FromPoint`, which is in half a dozen places spread over 5 files.
Might be worth considering, but not a blocker for this PR, in my opinion.